### PR TITLE
Confluence 0-Day IOCs

### PR DESCRIPTION
**Pull request by wey-chiang**
### Background

Adds IP IOCs for the recent Atlassian Confluence on-prem 0-Day

### Changes

* Added IOCs published by Volexity
* Added a new rule that checks for these IOCs

### Testing

* `make test`

### CODE REVIEW

File: panther_ioc_rules/atlassian_confluence_ip_iocs.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ExpectedResult: false
+    Log:
+      {
+        "dstport": 53,
+        "dstaddr": "1.1.1.1",
+        "srcaddr": "10.0.0.1",
+        "p_any_ip_addresses": ["1.1.1.1"],
+      }
+  -
+    Name: Log4j Indicator of Compromise (IP) Detected
```
Comment (nhakmiller) [inline panther_ioc_rules/atlassian_confluence_ip_iocs.yml:50]: I think copied over `Log4j`

File: panther_ioc_rules/atlassian_confluence_ip_iocs.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from panther_iocs import VOLEXITY_CONFLUENCE_IP_IOCS, ioc_match
+
+
+def rule(event):
+    return any(ioc_match(event.get("p_any_ip_addresses"), VOLEXITY_CONFLUENCE_IP_IOCS))
+
+
+def title(event):
+    ips = ",".join(ioc_match(event.get("p_any_ip_addresses"), VOLEXITY_CONFLUENCE_IP_IOCS))
+    return f"IP seen in May 2022 exploiting Confluence 0-Day: {ips}"
```
Comment (nhakmiller) [inline panther_ioc_rules/atlassian_confluence_ip_iocs.py:10]: The phrasing here seems a little awkward, maybe

```suggestion
    return f"IP seen from May 2022 exploitation of Confluence 0-Day: {ips}"
```

With phrasing it kind be very subjective so feel free to disregard this comment.

**Review by nhakmiller — APPROVED** [review]
Two small comments, otherwise looks good to me.

File: panther_ioc_rules/atlassian_confluence_ip_iocs.yml (hunk 1 above)
Comment (wey-chiang) [reply panther_ioc_rules/atlassian_confluence_ip_iocs.yml:50]: Nice catch, fixed

**Review by nhakmiller — APPROVED** [review]

**Pull request merged**